.ci/install_ksm_throttler.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+cidir=$(dirname "$0")
+
+source "${cidir}/lib.sh"
+
+clone_build_and_install "github.com/kata-containers/ksm-throttler"
+
+# If we are running under the metrics CI system then we do not want the proxy
+# to be dynmaically changing the KSM settings under us - we need control of them
+# ourselves
+if [[ ! $METRICS_CI ]]; then
+	systemctl enable vc-throttler.service
+	systemctl start vc-throttler.service
```

Comment:
I made a local test and `ksm-throttler` service is started by the `vc-throttler` service, so I think we do not need to start via the script.

Here a proof:
```
cc-tester@singlevm:~$ sudo systemctl status vc-throttler.service 
● vc-throttler.service - Virtcontainers based KSM throttling
   Loaded: loaded (/lib/systemd/system/vc-throttler.service; disabled; vendor preset: enabled)
   Active: inactive (dead)
     Docs: https://github.com/kata-containers/ksm-throttler
cc-tester@singlevm:~$ sudo systemctl status ksm-throttler.service 
● ksm-throttler.service - KSM throttling daemon
   Loaded: loaded (/lib/systemd/system/ksm-throttler.service; disabled; vendor preset: enabled)
   Active: inactive (dead)
     Docs: https://github.com/kata-containers/ksm-throttler
cc-tester@singlevm:~$ sudo systemctl start vc-throttler.service 
cc-tester@singlevm:~$ sudo systemctl status vc-throttler.service 
● vc-throttler.service - Virtcontainers based KSM throttling
   Loaded: loaded (/lib/systemd/system/vc-throttler.service; disabled; vendor preset: enabled)
   Active: active (running) since Tue 2018-02-20 18:07:15 UTC; 3s ago
     Docs: https://github.com/kata-containers/ksm-throttler
 Main PID: 1866 (vc)
    Tasks: 4
   Memory: 7.6M
      CPU: 10ms
   CGroup: /system.slice/vc-throttler.service
           └─1866 /usr/libexec/ksm-throttler/trigger/virtcontainers/vc -log debug

Feb 20 18:07:15 singlevm systemd[1]: Started Virtcontainers based KSM throttling.
Feb 20 18:07:15 singlevm vc[1866]: time="2018-02-20T18:07:15Z" level=debug msg="Waiting for /var/run/virtcontainers"
cc-tester@singlevm:~$ sudo systemctl status ksm-throttler.service 
● ksm-throttler.service - KSM throttling daemon                                                                                                                                          "singlevm" 16:10 20-Feb-18
   Loaded: loaded (/lib/systemd/system/ksm-throttler.service; disabled; vendor preset: enabled)
   Active: active (running) since Tue 2018-02-20 18:07:15 UTC; 6s ago
     Docs: https://github.com/kata-containers/ksm-throttler
 Main PID: 1867 (ksm-throttler)
    Tasks: 6
   Memory: 6.6M
      CPU: 11ms
   CGroup: /system.slice/ksm-throttler.service
           └─1867 /usr/libexec/ksm-throttler/ksm-throttler -log debug

Feb 20 18:07:15 singlevm systemd[1]: Started KSM throttling daemon.
Feb 20 18:07:15 singlevm ksm-throttler[1867]: time="2018-02-20T18:07:15Z" level=info name=ksm-throttler pid=1867 source=throttler version=0.0.1-3647ce2aba865dc877a390bde78ce27fa60a6903
Feb 20 18:07:15 singlevm ksm-throttler[1867]: time="2018-02-20T18:07:15Z" level=debug msg="Starting KSM throttling service at /var/run/ksm-throttler/ksm.sock" name=ksm-throttler pid=1867 source=throttler

```

Reply:
Hi @chavafg - I checked the `systemd` units themselves and you are correct - we only need `vc`.